cloudbuild/unit_test_jobs.jsonnet

```diff
@@ -21,20 +22,28 @@ local unittest = base.BaseTest {
   entrypoint: [
     'bash',
     '-c',
-    |||
-      # Build custom ops from source
-      python build_deps/configure.py
-      bazel-5.4.0 build keras_cv/custom_ops:all --verbose_failures
-      cp bazel-bin/keras_cv/custom_ops/*.so keras_cv/custom_ops/
-      export TEST_CUSTOM_OPS=true
+    std.format(
+      |||
+        export KERAS_BACKEND=%s
+        export JAX_ENABLE_X64=true
```

Comment:
Why do you end up needed int64/float64? Just curious really.

In KerasNLP we have just been trying to go int32 everywhere by default.

Reply:
I think we probably _should_ just do int32 everywhere. During NMS porting I had some things that were real sticklers about int64 in TF and I never found a way to work around

Reply:
Could we potentially just change the dtype for those tests? Asking since we're always flirting with OOM issues and int64 can't help

Reply:
Potentially, yes. I did some looking and it seems like the only place we're using int64 internally with KerasCore is in the YOLOV8 label encoder -- I guess I already got rid of them from NMS. I'll check with Tirth if he's planning on getting rid of those.